src/mol-canvas3d/passes/ssao.ts

```diff
@@ -681,14 +682,15 @@ function getBlurKernel(kernelSize: number): number[] {
     return kernel;
 }
 
+const pcg = new PCG();
 const RandomHemisphereVector: Vec3[] = [];
 for (let i = 0; i < 256; i++) {
     const v = Vec3();
-    v[0] = Math.random() * 2.0 - 1.0;
-    v[1] = Math.random() * 2.0 - 1.0;
-    v[2] = Math.random();
+    v[0] = pcg.float() * 2.0 - 1.0;
+    v[1] = pcg.float() * 2.0 - 1.0;
+    v[2] = pcg.float();
     Vec3.normalize(v, v);
-    Vec3.scale(v, v, Math.random());
+    Vec3.scale(v, v, pcg.float());
     RandomHemisphereVector.push(v);
```

Comment:
Based on what @giagitom points out it seems like this sphere sampling isn't very uniform.

It seems like the first 3*256 numbers with this particular seed either (or a combination of):
- don't produce a very good distribution of vectors 
- the way you are generating a random vector on a sphere is biased. I think you would get a better distribution if you generated two angles in spherical coordinates and converted that to a vector instead. (or see say https://math.stackexchange.com/questions/1585975/how-to-generate-random-points-on-a-sphere; PCG produces uniform distribution, someone mentions gaussian instead needed for this). Basically turning uniform distribution into sphere points requires more transformation than just a normalization.